src/libstd/sys/common/remutex.rs

```diff
@@ -205,6 +204,7 @@ mod tests {
     }
 
     #[test]
+    #[allow(unused_variables)]
```

Comment:
You can just rename to `let _foo = expr` in this test to avoid adding the attribute